src/WaterPokemon.java

```diff
@@ -0,0 +1,50 @@
+public class WaterPokemon extends Pokemon {
+    private String type = "Water \uD83C\uDF0A";
+    private String fastAttack;
+    private String chargedAttack;
+
+    public WaterPokemon(String name, int level, int hp, String fastAttack, String chargedAttack) {
+        super(name, level, hp);
+        this.fastAttack = fastAttack;
+        this.chargedAttack = chargedAttack;
+    }
+
+    @Override
+    public void fastAttack() {
```

Comment:
je @Overwrite je methodes in al je subclasse met de zelfde SOUT, de bedoeling van een @Overwrite is dat de implementatie per subclasse verchilt. Je kunt nu net zo goed deze SOUT in je Abstracte klassen plaatsen. Nu herhaal je jezelf 4x.

Reply:
Zie ook comment in Pokemon class.

En dan hier doet je super.fastAttack(this.fastAttack);